src/main/com/marginallyclever/makelangelo/paper/PaperMenuItem.java

```diff
@@ -16,10 +16,12 @@ public PaperMenuItem(PaperSettings paperSettings, PaperSize paperSize) {
         });
     }
 
-    public void updateSelected() {
+    public boolean updateSelected() {
         double w = paperSettings.getCurrentPaper().getPaperWidth();
         double h = paperSettings.getCurrentPaper().getPaperHeight();
-        setSelected(paperSize.width == w && paperSize.height == h);
+        boolean selected = paperSize.width == w && paperSize.height == h;
+        setSelected(selected);
+        return selected;
```

Comment:
is this a setter and a getter rolled into one?

Reply:
it is not intended, the name of the method `updateSelected()` in neither a setter nor a getter. I'll rename the boolean